openedx_ledger/admin.py

```diff
@@ -90,6 +93,52 @@ class Meta:
         readonly_fields = _all_fields
     inlines = [ExternalTransactionReferenceInlineAdmin]
 
+    change_actions = ('unenroll_with_refund', 'unenroll',)
+
+    def unenroll_with_refund(self, request, obj):
+        """
+        Redirect to the unenroll view.
+        """
+        # url names coming from get_urls are prefixed with 'admin' namespace
+        manage_learners_url = reverse("admin:unenroll_with_refund", args=(obj.uuid,))
+        return HttpResponseRedirect(manage_learners_url)
+
+    unenroll_with_refund.label = "Unenroll & Refund"
+    unenroll_with_refund.short_description = (
+        "Reverse a transaction and unenroll the learner from the platform representation of the course."
+    )
+
+    def unenroll(self, request, obj):
+        """
+        Redirect to the unenroll view.
+        """
+        # url names coming from get_urls are prefixed with 'admin' namespace
+        manage_learners_url = reverse("admin:unenroll", args=(obj.uuid,))
+        return HttpResponseRedirect(manage_learners_url)
+
+    unenroll.label = "Unenroll"
+    unenroll.short_description = (
+        "Unenroll the learner from the platform representation of the course without reversing the transaction object."
+    )
+
+    def get_urls(self):
+        """
+        Returns the additional urls used by the custom object tools.
```

Comment:
question: what are "custom object tools"?

Reply:
custom objects refers to `DjangoObjectActions` in this case

Reply:
I can update the wording to be more clear